docs/02-guides/server-side-events/integrations/google-analytics.mdx

```diff
@@ -0,0 +1,88 @@
+configuration:
+
+```ts title="app/config/analytics.ts"
+//...
+export default {
+  analytics: {
+    enable: true,
+    debug: process.env.NODE_ENV !== "production",
+    // highlight-start
+    serverTracking: {
+      enabled: true,
+    },
+    // highlight-end
+    //..
+  } satisfies AnalyticsConfig,
+};
+```
+
+And that's it! Now every event will be submitted to Google Analytics by your
+server!
+
+## Compatible events
+
+| Front-Commerce event        | Google Analytics event                                                                                                                  |
+| --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| `Products Searched`         | [search](https://developers.google.com/analytics/devguides/collection/protocol/ga4/reference/events#search)                             |
+| `Product List Viewed`       | [view_item_list](https://developers.google.com/analytics/devguides/collection/protocol/ga4/reference/events#view_item_list)             |
+| `Product Clicked`           | [select_content](https://developers.google.com/analytics/devguides/collection/protocol/ga4/reference/events#select_content)             |
+| `Product Viewed`            | [view_item](https://developers.google.com/analytics/devguides/collection/protocol/ga4/reference/events#view_item)                       |
+| `Product Added`             | [add_to_cart](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?client_type=gtag#add_to_cart)           |
+| `Product Removed`           | [remove_from_cart](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?client_type=gtag#remove_from_cart) |
+| `Cart Viewed`               | [view_cart](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?client_type=gtag#view_cart)               |
+| `Checkout Started`          | [begin_checkout](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?client_type=gtag#begin_checkout)     |
+| `Payment Info Entered`      | [add_payment_info](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?client_type=gtag#add_payment_info) |
+| `Order Completed`           | [purchase](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?client_type=gtag#purchase)                 |
+| `User Logged in`            | [login](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?client_type=gtag#login)                       |
+| `User Created`              | [sign_up](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?client_type=gtag#sign_up)                   |
+| `Product Added to Wishlist` | [add_to_wishlist](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?client_type=gtag#add_to_wishlist)   |
```

Comment:
**note:** I couldn't test any of those link at the time of reviewing, as I was getting "Service unavailable" errors. I trust you ensured those are right.

![image](https://github.com/front-commerce/developers.front-commerce.com/assets/4162177/0563f6bf-3955-4e33-975a-14c44cc680c5)

Reply:
Tried it right now and it seems to work well:
![image](https://github.com/front-commerce/developers.front-commerce.com/assets/6305536/90aae010-675b-47d7-b340-18a5704d9d16)

Reply:
Thanks!